src/content/en/updates/2018/06/feature-policy.md

```diff
@@ -0,0 +1,336 @@
+target and good reference docs will definitely be needed.
+
+For now, there are a couple of ways to see what features are controllable.
+
+1. Check out our [Feature Policy Kitchen Sink][sink] of demos. It has examples
+of each policy that's been implemented in Blink.
+-  Check [Chrome's source](https://cs.chromium.org/chromium/src/third_party/blink/renderer/platform/feature_policy/feature_policy.cc?g=0&l=138) for the list of feature names.
+-  If you have the `--enable-experimental-web-platform-features` flag on in `chrome:flags`, query `document.policy.allowedFeatures()` find the list:
+
+        geolocation
+        midi
+        camera
+        usb
+        magnetometer
+        fullscreen
+        animations
+        payment
+        picture-in-picture
+        accelerometer
+        vr
+        ...
+
+- Check [chromestatus.com][chromestatusfilter] for the policies that have been implemented or being considered in Blink.
+
+To determine _how_ to use some of these policies, check out the
+[spec's Github repo](https://github.com/WICG/feature-policy/tree/master/policies).
+It contains a few explainers on some of the policies.
+
+## FAQ
+
+#### When do I use Feature Policy? {: #whentouse }
+
+Since all policies are opt-in, use enable policies when/where they makes sense.
+For example, if your app is an image gallery, it would be logical to use
+the `maximum-downscaling-image` policy to make sure you're not pushing
+down images that are far too big on mobile.
+
+Other policies like `document-write` and `sync-xhr` are generally good policies
+to always enable. Since these APIs are considered harmful to web performance,
+you can use Feature Policy as a gut check that your app never uses them!
+
+#### Do I use Feature Policy in development or production?  {: #devorprod }
+
+Both. We recommend turning policies on during development keeping the policies
+active while in production.
+
+Turning policies on during development can help you start off on the right
+track. It'll help you catch any unexpected regressions before they happen.
+
+Keeping policies on for production gives your users a guarantee that the UX
+won't change from under them.
+
+#### What are the inheritance rules for iframe content? {: #inheritancerules }
+
+- All feature policies are opt-in. You control what features to
+enable/modify/disable in your app.
+- Scripts (either first or third-party) inherit the policy of their browsing
+context. That means that top-level scripts inherit the main document's policy.
+- `iframe` content inherits the more strict policy between the embedding page
+and the `allow` list on the frame itself.
+See [the `allow` attribute on iframes](#iframe).
+
+#### What browsers support Feature Policy? {: #support }
+
+As of now, Chrome is the only browser to support feature policy. However,
+since the entire API surface is opt-in or feature dectable, **Feature Policy
+lends itself nicely to progressive enhancement**.
+
```

Comment:
Seems like this increases security, but you're completely silent on that. For instance if someone injects `document.write()` into a page, it will fail if it's disallowed by feature policy. Is there any reason you haven't mentioned this in the article?

Reply:
I'm not sure we should make that claim. One could draw the conclusion about similar things like `geolocation` and privacy.

There may be some feature policies (now or in the future) that you could classify as being helpful for security, but CSP is really the proper tool for tightening the security of an app.